github_api_helper.js

```diff
@@ -31,10 +31,11 @@ function inProgressAPIresponse (context, headSha) {
  * Get Precaution config file contents as raw data,
  * if that file exists.
  * @param {import('probot').Context} context Probot context
+ * @param {String} customConfigPath Optional (default value config.configFilePath)
```

Comment:
If this parameter will only ever be overridden by our tests I think it's an indicator of a deficiency in our test setup.

I assume this is because we don't actually have a git repo with expected files, we're mocking a repo? Is it worth writing a generic function for our tests that can populate a local git repo with some files, then we can have tests create the repo in setup and delete it in teardown?

Note: this is a genuine question, not me saying we should do this.

Reply:
Right now we are mocking GitHub simulating their responses and behavior as much as we could. 
We use hardcoded responses given values and files to verify that our mocked GitHub version returns the result we expect.

Reply:
Understood. My suggestion may be too much work as repo contents aren't really what we care about testing. The main thing I wanted to note is that we're adding function parameters just to support testing and that doesn't quite _feel_ right. Something to keep in mind, perhaps.